minishift_sanity.py

```diff
@@ -92,90 +92,164 @@ def test_ms_start(self):
         else:
             self.fail("Minishift start failed")
             provision_path = None
+    """
     
-    """        
+    def test_ms_setup_cdk(self):
```

Comment:
Added Is_Downstream boolean variable into yaml config. If set to true, test for downstream setup-cdk command will run, if not it will be marked passed and rest of test is skipped.